pkg/kubectl/genericclioptions/name_flags.go

```diff
@@ -40,6 +40,10 @@ func (f *NamePrintFlags) Complete(successTemplate string) error {
 	return nil
 }
 
+func (f *NamePrintFlags) AllowedFormats() []string {
+	return []string{"name", `""`}
```

Comment:
I'd rather we don't say empty is allowed format.

Reply:
Done.